nylas/resources/messages.py

```diff
@@ -85,24 +123,99 @@ def update(
             path=f"/v3/grants/{identifier}/messages/{message_id}",
             response_type=Message,
             request_body=request_body,
-            query_params=query_params,
         )
 
-    def destroy(
-        self, identifier: str, message_id: str, query_params: Optional[Dict] = None
-    ) -> DeleteResponse:
+    def destroy(self, identifier: str, message_id: str) -> DeleteResponse:
         """
         Delete a Message.
 
         Args:
             identifier: The identifier of the grant to delete the message for.
             message_id: The identifier of the message to delete.
-            query_params: The query parameters to include in the request.
 
         Returns:
             The deletion response.
         """
         return super(Messages, self).destroy(
             path=f"/v3/grants/{identifier}/messages/{message_id}",
-            query_params=query_params,
         )
+
+    def send(
+        self, identifier: str, request_body: SendMessageRequest
+    ) -> Response[Message]:
+        """
+        Send a Message.
+
+        Args:
+            identifier: The identifier of the grant to send the message for.
+            request_body: The request body to send the message with.
+
+        Returns:
+            The sent message.
+        """
+        json_response = self._http_client._execute(
+            method="POST",
+            path=f"/v3/grants/{identifier}/messages/send",
+            data=_build_form_request(request_body),
+        )
+
+        return Response.from_dict(json_response, Message)
+
+    def list_scheduled_messages(
+        self, identifier: str
+    ) -> Response[ScheduledMessagesList]:
+        """
+        Retrieve your scheduled messages.
+
+        Args:
+            identifier: The identifier of the grant to delete the message for.
+
+        Returns:
+            Response: The list of scheduled messages.
+        """
+        json_response = self._http_client._execute(
+            method="GET",
+            path=f"/v3/grants/{identifier}/messages/schedules",
+        )
+
+        return Response.from_dict(json_response, ScheduledMessagesList)
+
+    def find_scheduled_messages(
+        self, identifier: str, schedule_id: str
+    ) -> Response[ScheduledMessage]:
+        """
+        Retrieve your scheduled messages.
+
+        Args:
+            identifier: The identifier of the grant to delete the message for.
+            schedule_id: The id of the scheduled message to retrieve.
+
+        Returns:
+            Response: The scheduled message.
+        """
+        json_response = self._http_client._execute(
+            method="GET",
+            path=f"/v3/grants/{identifier}/messages/schedules/{schedule_id}",
+        )
+
+        return Response.from_dict(json_response, ScheduledMessage)
+
+    def stop_scheduled_messages(
```

Comment:
same here—`stop_scheduled_message`?

Reply:
You would be correct, again! Will also fix it, thank you :)